### PR TITLE
Lavaland Seed Vault - fixing oversights.

### PR DESCRIPTION
**Pull request by Venuska1117**
## What Does This PR Do
Fixes Cryopod sending messages to station when diona decide to cryo, replace wooden table with internals crate with all stuff necessery to go outside exploring, updated "seed vault objectives" to contain information about new stuff in "internals crate" and added new diona corpse with contain "seed vault" stuff on it, like uniform, pickaxe and note with informations.

## Why It's Good For The Game
Fix my mistake of forgetting to add any pointers "how to get oxygen" for dionas, so they can spread grass, now you have crate with 4 sets of internals and corpse with note expanding "lore" a little.

(and change cryopods to offstation so they dont get informed when someone cryo as diona)

## Images of changes
_Thanks Henri215_
image

## Testing
Run server, spawned ruin, checked if everything is on place, don't get any errors.

## Changelog
:cl: Venuska1117
tweak: Now seed vault cryopods don't send message to station.
tweak: Fixed dionas don't have any way to explore outside without suffocation, now they have internals crate with emergency internals.
add: Diona corpse near ship to give some pointers for newbies if they decide to explore further away from ship.
/:cl:

### CODE REVIEW

File: code/modules/paperwork/paper.dm [hunk 1]
```diff
@@ -560,6 +560,10 @@
 /obj/item/paper/crumpled/bloody
 	icon_state = "scrap_bloodied"
 
+/obj/item/paper/crumpled/bloody/ruins/lavaland/seed_vault/discovery
+	name = "discoveries and toughts"
```
Comment (S34NW) [inline code/modules/paperwork/paper.dm:564]:
```suggestion
	name = "discoveries and thoughts"
```

File: code/modules/paperwork/paper.dm [hunk 2]
```diff
@@ -560,6 +560,10 @@
...
...e great we were making plants and discovering new ones over time, we spread 'special' grass around our ship, it gone great, over time we started lack oxygen in our tanks to continue spreading of grass, so we started to mutate plans in way to discover ability to breathe little longer in low oxygen enviornement, after trails of fails and success we discovered a few species of plants with can be usefull like healing plants, addiction plants, glowing shrooms with are usefull for us so we don't die from lack of light, and way more plants, but there was one with brough most of our attention, it may not give us oxygen but allowed us to breathe in low oxygen enviornement, we called it 'space tobacco' its poisonous but it contain Salbutamol chemical with allow you to breathe in low oxygen enviornement when consumed, only downsite is you need extract bad chemicals away just for this one, after we did it, we started spreading grass but after some travel i got attacked by strange creature and got chased, i wandered too far from ship and my storage of plant is running out like my ability to breathe for longer, i am starting to suffocate, i need find my way back quickly before i will suffocate."
```
Comment (S34NW) [inline code/modules/paperwork/paper.dm:565]:
```suggestion
	info = "As the Diona species, we awoke aboard our terraformation vessel with the primary goal of reshaping the alien world. Our endeavors were highly successful, as we cultivated various plant species and made astonishing discoveries throughout our journey. We seeded a remarkable 'special' grass around our ship, which thrived splendidly. However, as time passed, we faced a growing challenge - a shortage of oxygen in our containment tanks hindered our ability to spread the grass further. In response, we embarked on a series of trials and experiments to engineer plants with the capacity to survive in low-oxygen environments, thus extending our breath of life. <br>Through a series of trials, combining failures and successes, we unveiled several plant species with unique attributes. Some proved to be valuable for healing purposes, while others offered addictive properties. Glowing mushrooms emerged as a source of vital light, preventing us from succumbing to the darkness. Among these discoveries, one plant commanded our utmost attention – the 'space tobacco.' While this species did not generate oxygen, it contained a chemical known as Salbutamol, enabling us to respire in low-oxygen conditions when consumed. The only drawback was the need to meticulously extract harmful compounds for its safe utilization. <br>Amid our efforts to expand the greenery, an unexpected encounter transpired. I found myself under assault by an enigmatic creature, and I was forced to flee in haste, straying too far from our vessel. As I stand now, my supplies of life-sustaining plants are dwindling, as is my ability to endure in this low-oxygen environment. Suffocation looms, and I must hasten my return to the safety of our ship to avert this dire fate."
```

This had a fair few errors in the original text, alongside grammatical issues and the fact it was one giant sentence.
I used AI to rewrite it into something more naturally English.

**Review by S34NW — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/paper.dm (hunk 2 above)
Comment (Venuska1117) [reply code/modules/paperwork/paper.dm:565]: God bless you, even grammar correction website stopped function Ing for me when i placed it in 😞

Comment (Venuska1117) [reply code/modules/paperwork/paper.dm:565]: I need to start using chat GPT

**Review by S34NW — APPROVED** [review]

**Review by Henri215 — APPROVED** [review]

**Pull request merged**